README.md

```diff
@@ -42,6 +42,9 @@ _Note: Some features are dependent on how much data is provided during event gat
 
 ## Usage
 
+Regardless of your deployment strategy, you may find reading the
+[Things to Know](#things-to-know) section prior to deployment.
```

Comment:
"...may find reading the Thing to Know section prior..."

-->

"...may find reading the Thing to Know section **helpful (??)** prior..."